.gitignore

```diff
@@ -6,3 +6,4 @@ html/
 *.pyc
 *.swp
 *.tmp
+src/
```

Comment:
This is a folder than is sometimes automatically generated by pip when you run:
pip install -r requirements.pip

We don't have any of our own code in this folder.